app/views/audit_mailer/summary.text.erb

```diff
@@ -0,0 +1,15 @@
+<%= _("Foreman audit summary") %>
+=================================
+<% if @audits.size > 0 %>
+<%= n_('Displaying %{num_audits} of %{total_audits} audit',
+       'Displaying %{num_audits} of %{total_audits} audits', @count) % {:num_audits => @limit, :total_audits => @count} %>
+
+<% @audits.each do |audit| %>
+<%= audit.username %> (<%= audit.remote_address %>) <%= audit_action_name audit%> <%= audited_type audit %>: <%= audit_title(audit) %> <%= audit_path(:id => audit, :host => @url.host, :port => @url.port, :only_path => false, :protocol => @url.scheme) %>
```

Comment:
Nitpick, put the URL in brackets or a newline perhaps, to separate it from the text of the audit entry